scripts/lanl-builds/cray_nvcc_gcc.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright (c) 2016-24, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+if [[ $# -lt 3 ]]; then
+  echo
+  echo "You must pass 3 arguments to the script (in this order): "
+  echo "   1) compiler version number for nvcc"
+  echo "   2) CUDA compute architecture (number only, not 'sm_90' for example)"
+  echo "   3) compiler version number for gcc"
+  echo
+  echo "For example: "
+  echo "    cray_nvcc_gcc.sh 12.5 90 12.3.0"
+  exit
+fi
+
+COMP_NVCC_VER=$1
+COMP_ARCH=$2
+COMP_GCC_VER=$3
+shift 3
+
+BUILD_SUFFIX=lanl_cray-nvcc${COMP_NVCC_VER}-${COMP_ARCH}-gcc${COMP_GCC_VER}
+
+echo
+echo "Creating build directory build_${BUILD_SUFFIX} and generating configuration in it"
+echo "Configuration extra arguments:"
+echo "   $@"
+echo
+
+rm -rf build_${BUILD_SUFFIX} >/dev/null
+mkdir build_${BUILD_SUFFIX} && cd build_${BUILD_SUFFIX}
+
+cmake \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DGCC_HOME=/usr/projects/hpcsoft/tce/24-07/cos3-aarch64-cc90/compilers/gcc/${COMP_GCC_VER} \
+  -DCMAKE_CUDA_ARCHITECTURES=${COMP_ARCH} \
+  -DCUDA_TOOLKIT_ROOT_DIR=/opt/nvidia/hpc_sdk/Linux_aarch64/24.7/cuda/${COMP_NVCC_VER} \
+  -C ../host-configs/lanl-builds/cray/nvcc_gcc_X.cmake \
+  -DENABLE_BENCHMARKS=On \
+  -DCMAKE_INSTALL_PREFIX=../install_${BUILD_SUFFIX} \
+  "$@" \
+  ..
+
+echo
+echo "***********************************************************************"
+echo
+echo "cd into directory build_${BUILD_SUFFIX} and run make to build RAJA"
+echo
+echo "  Please note that you have to disable CUDA GPU hooks when you run"
+echo "  the RAJA tests; for example,"
+echo
+echo "    make test"
```

Comment:
Was there supposed to be something else in the example here apart from `make test`?

Reply:
Probably. Our other nvcc scripts have something like this: https://github.com/LLNL/RAJA/blob/develop/scripts/lc-builds/blueos_nvcc_gcc.sh#L63